src/staking-router/vetted-keys.ts

```diff
@@ -0,0 +1,16 @@
+import { RegistryKey } from 'keys-api/interfaces/RegistryKey';
+import { RegistryOperator } from 'keys-api/interfaces/RegistryOperator';
+
+export function getVettedUnusedKeys(
```

Comment:
perhaps we should write a small test for this function?

Reply:
yes, will do